### PR TITLE
fix(system-backup): NPE when PV source not from CSI

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/5286

### CODE REVIEW

**Comment by c3y1huang** [comment]
https://drone-pr.longhorn.io/longhorn/longhorn-manager/4049/1/2
cc @yangchiu

**Review by shuo-wu — APPROVED** [review]

File: controller/kubernetes_pv_controller_test.go [hunk 1]
```diff
@@ -61,53 +61,57 @@ func (tc *KubernetesTestCase) copyCurrentToExpect() {
...
 		},
-		Status: apiv1.PersistentVolumeStatus{
-			Phase: apiv1.VolumeBound,
+		Status: corev1.PersistentVolumeStatus{
+			Phase: corev1.VolumeBound,
+		},
+	}
+}
+
+func newPVSourceCSI() *corev1.CSIPersistentVolumeSource {
```
Comment (innobead) [inline controller/kubernetes_pv_controller_test.go:90]: Actually you can add this as `newPVSourceCSI`, but return `corev1.PersistentVolumeSource` instead to save repeated the outside structure `PersistentVolumeSource`.

**Review by innobead — COMMENTED** [review]
LGTM. 

nit: shoud rename `TestReconcileRolloutSave` to `TestReconcileSystemBackup`.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@Mergifyio backport v1.4.x

**Pull request merged**